ext/drmgr/drmgr.c

```diff
@@ -1513,12 +1512,13 @@ drmgr_modload_event(void *drcontext, const module_data_t *info,
             continue;
         bool is_using_user_data = iter.cbs.generic[i].is_using_user_data;
         void *user_data = iter.cbs.generic[i].user_data;
-        if (is_using_user_data == false)
+        if (is_using_user_data == false){
```

Comment:
nit: space before {

Reply:
@derekbruening One thing that you might not be aware of. Commit 613ff1a, of this PR, resulted in a failed build. I am not sure what caused the issue, but the test seems to unreliably fail or pass. It is not a test which seems to check my PR.

Reply:
Ill get the space in; not a prob.

Reply:
Looking: ok it was tool.drcacheoff.burst_threads which is a known flaky test (#2092) nobody has yet marked flaky or disabled or tried to fix.

Reply:
Done.